src/scripts/generateBridgeList.ts

```diff
@@ -34,15 +46,8 @@ type Extensions = {
  * @param outputFilePath
  * @param tokensToReplace
  */
-export async function generateBridgedList(
-  chainId: SupportedChainId,
-  tokenListSource: string | TokenList,
-  bridgeContractAddress: string,
-  bridgeContractAbi: ContractInterface,
-  methodName: string,
-  outputFilePath: string,
-  tokensToReplace: Record<string, string | null> = {},
-): Promise<void> {
```

Comment:
it had too many arguments, so I decided to change from positional parameters to named parameters